src/pages/settings/AboutPage.js

```diff
@@ -0,0 +1,175 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'lodash';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import CONST from '../../CONST';
+import {
+    Link, Eye, MoneyBag, Bug, NewWindow,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {
+    withLocalizePropTypes,
+} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import MenuItem from '../../components/MenuItem';
+import Logo from '../../../assets/images/expensify-cash.svg';
+import {version} from '../../../package.json';
+import openURLInNewTab from '../../libs/openURLInNewTab';
+import ONYXKEYS from '../../ONYXKEYS';
+
+
+const propTypes = {
+    // All reports shared with the user
+    reports: PropTypes.shape({
+        reportID: PropTypes.number,
+        reportName: PropTypes.string,
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+
+};
+
+const AboutPage = ({translate, reports}) => {
+    const [reportID, setreportID] = useState(null);
+
+    useEffect(() => {
+        const reportKey = _.findKey(reports, ['lastActorEmail', CONST.EMAIL.CONCIERGE]);
+        if (reportKey) {
+            setreportID(reports[reportKey].reportID);
+        }
+    }, [reports]);
+
+
+    const menuItems = [
+        {
+            translationKey: 'initialSettingsPage.aboutPage.appDownloadLinks',
+            icon: Link,
+            action: () => {
+                Navigation.navigate(ROUTES.SETTINGS_APP_DOWNLOAD_LINKS);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewTheCode',
+            icon: Eye,
+            iconRight: NewWindow,
+            action: () => {
+                openURLInNewTab(CONST.GITHUB_URL);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.viewOpenJobs',
+            icon: MoneyBag,
+            iconRight: NewWindow,
+            action: () => {
+                openURLInNewTab(CONST.UPWORK_URL);
+            },
+        },
+        {
+            translationKey: 'initialSettingsPage.aboutPage.reportABug',
+            icon: Bug,
+            action: () => {
+                if (reportID) { Navigation.navigate(ROUTES.getReportRoute(reportID)); }
```

Comment:
@shawnborton over here

Reply:
Ah okay, in the video you posted it looks like it was going to a new window. Perhaps you just need to update the screenshots/videos in your original comment.